whitepaper/chapter01.md

```diff
@@ -44,13 +44,15 @@ There are some additional key principles to cloud native that are important to g
 <a name="1.3.1"></a>
 ### 1.3.1 What "loosely coupled" means in cloud native systems
 
-Cloud native systems have a clear separation between the microservices. The separation is based on a technology like containers which allows independent life cycle management of each microservice. Cloud native applications, which are built on these microservices, and systems generally benefit from segregating as many services as possible into different containers. Additional benefits of cloud native architectures include scalability (vertical and horizontal) per microservice. This simplifies deployment in a wide variety of infrastructure environments, ranging from endpoint device to edge station to large data centres. It also allows service providers to deploy resources in the most efficient manner possible while maintaining necessary service levels for regulated and unregulated service types.
+Cloud native systems have a clear separation between the microservices. The separation is based on a technology like containers which allows independent life cycle management of each microservice. Cloud native applications, which are built on these microservices, and systems generally benefit from design considerations that segregate services where appropriate by leveraging technology such as containers. Additional benefits of cloud native architectures include scalability (vertical and horizontal) per microservice. This simplifies deployment in a wide variety of infrastructure environments, ranging from endpoint device to edge station to large data centres. It also allows service providers to deploy resources in the most efficient manner possible while maintaining necessary service levels for regulated and unregulated service types.
```

Comment:
> design considerations that segregate services where appropriate by leveraging technology such as containers.

It seems like _where appropriate_ is too vague. Can we leverage something from the [cloud native microservices principles](https://networking.cloud-native-principles.org/cloud-native-microservice-principles) to expand or reword this?